poetry.lock

```diff
@@ -0,0 +1,936 @@
+[[package]]
```

Comment:
why do we need this file in vcs?

Reply:
To eliminate the problem with versions of dependencies for people developing the library. 
`You should commit the poetry.lock file to your project repo so that all people working on the project are locked to the same versions of dependencies (more below).` - from docs of Poetry https://python-poetry.org/docs/basic-usage/#installing-without-poetrylock

Reply:
This is not a "project" project, though, it's a library.
Pinning all dependency versions may mask any possible incompatibilities from lib developers.
Top level dependency version pinning  in project.toml should be enough, as far as I understand.